.github/workflows/c.yaml

```diff
@@ -252,7 +260,8 @@ jobs:
   windows_build:
     name: Windows build
     needs: windows_setup
-    uses: openslide/openslide-winbuild/.github/workflows/windows.yml@main
+    uses: jcupitt/openslide-winbuild/.github/workflows/windows.yml@add-libdicom-prototype
```

Comment:
This is causing the workflow not to start:

```
error parsing called workflow
".github/workflows/c.yaml"
-> "jcupitt/openslide-winbuild/.github/workflows/windows.yml@add-libdicom-prototype"
: workflow was not found. See https://docs.github.com/actions/learn-github-actions/reusing-workflows#access-to-reusable-workflows for more information.
```

Reply:
ooop, thanks! I thought `main` was a branch name.